frontend/src/employees/TableListOfEmployees.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import useUser from '../Hooks/useUser';
+
+export default function TableListOfEmployees() {
+  const { user } = useUser();
+
+  return (
+    <div className="border border-secondary">
+      <div className="container-fluid">
+        <table className="table">
+          <thead>
+            <tr>
+              <th scope="col">#</th>
+              <th scope="col">Név</th>
+              <th scope="col">E-mail</th>
+              <th scope="col">Aktiv</th>
+            </tr>
+          </thead>
+          <tbody>
+            {user ? (
+              user.map(user => (
```

Comment:
Egyes szám, többes szám? Ha a useUser egy listát ad vissza, akkor legyen inkább useUsers és adjon vissza users-t, a map-en belül meg egy user van, akkor az maradjon user.

Reply:
De az objektumban továbbra is user van, nem users, ez így elég megtévesztő, mert itt a user és a user nem ugyanaz.